compiler/luci-interpreter/src/loader/KernelBuilder.h

```diff
@@ -39,6 +39,7 @@ class KernelBuilder : public luci::CircleNodeVisitor<std::unique_ptr<Kernel>>
   {
   }
 
+  std::unique_ptr<Kernel> visit(const luci::CircleNode *) override;
```

Comment:
although implementation doesn't use argument, could you please add arg name `node` here to match like other `visit()` ?